### ISSUE TITLE
Add Motor function

### ISSUE THREAD

**Issue by bosforox** [dfki-ric/phobos#212]
**Describe the bug**

- It is possible to get a motor (and controller, if this option also selected) for a "fixed" link  (like a root) in set of block selected links.(phobos > hardware >add motor) 
-To activate the "Add Motor" function, it is necessary to first select only "one" link. This function cannot be activated if you make a block selection (B+Mouse-Left) for the multiple obj selection at the beginning. However, it is possible to add a block selection if this function was already activated in the previous step. But if you have also selected a fixed link, this function automatically adds a motor (and a controller, if this option selected) in to this fixed link. It may help if the phobos property of the joint is checked first and if it is not a fixed joint then the motor can be added. 

** Additional points **

- By adding motors, the name of motors can be automatically generated in the name.00x format. This can lead to other naming problems, such as composite model generation by Q-Rock. Perhaps it can help generally to user to add the link name as prefix and the motor name as subfix. 
- With the model import function (e.g. urdf import) the model parameters (e.g. maximum speed and effort of a joint) are automatically integrated as link parameters which are also related to the motor parameter. When creating motors, these parameters may also be saved as motor parameters.  (@kavonszadkowski please look at phobos-schrips repo> qrock branch>Qrock_editControllerMotorParam)

@2maz

**Comment by hwiedDFKI** [comment]
Yes, batch editing for multiple objects is not yet in depth supported. I consider this as feature.
